src/ficamp/__main__.py

```diff
@@ -39,6 +40,10 @@ def cli() -> argparse.Namespace:
     )
     categorize_parser.set_defaults(func=categorize)
 
+    # Subparser for the sync command
+    categorize_parser = subparsers.add_parser("sync", help="Report transactions")
```

Comment:
I would call the command "sink" with help: "send or store the results to the chosen sink" or something like that